assets/hip-1056/protobuf/stream/output/smart_contract_service.proto

```diff
@@ -129,33 +103,119 @@ message ContractSystemDeleteOutput {}
+      */
+     int32 log_index = 2;
+
+     /**
+      * The Log data
+      */
+     bytes data = 3;
+
+     /**
+      * The logc topics
+      */
+     repeated bytes topics = 4;
+}
+
+/**
+ * EVM tranactin execution output details
+ * Details maps to the log eth_getTransactionReceipt response without repeating info already available in the input transaction
+ */
+message EVMTransactionOutput {
+     /**
+      * A bloom filter produced by this contract call.<br/>
+      * Ethereum uses this bloom filter to search for call results in the
+      * Ethereum block history. High false positive rates make the bloom
+      * filters quite limited value.
+      */
+     bytes bloom = 1;  
+
+     /**
+      * Result data from the function call.
+      * <p>
+      * This SHALL be encoded in RLP bytecode format.
+      */
+     bytes contractCallResult = 2;
+
+     /**
+      * Any error message produced by the contract call.
+      * <p>
+      * This SHALL be unset if the contract call succeeded.
+      */
+     string errorMessage = 3;
+
+     /**
+      * A quantity of "gas" used.<br/>
+      * This represents the resource units expended to execute this
+      * contract call, and correlates to transaction costs.
+      */
+     uint64 gas_used = 4;
+
+     /**
+      * Any Log events produced by this contract call.
+      */
+     repeated EVMTransactionLog logs = 5;
+}
+
+/**
+ * The slots read during the contract execution. These slots may or may not have been overwritten.
+ * The associated writes and contract will be found in the state changes MapUpdateChange KVs
+ * Q: Should we add the ContractId in here for simplicity?
+ */
+message SlotRead {
+     /**
+      * The contract storage slot counter used to correlate slots across BlockItems for a given contract in a transactions.
+      */
+     int32 index = 1;
+	
+     /**
+      * The storage value in this slot, may be left-padded with zeros to form a 256-bit word.
+      */
+     bytes read_value = 2;
+}
+
+/**
+ * EVM tranaction execution trace details
+ * Details maps to the variable needed for debugging executions
+ */
+message EVMTraceData {
     /**
-     * An ethereum hash value.
-     * <p>
-     * This SHALL be a keccak256 hash of the ethereumData.
+     * The inter contract interaction details. This represents the internal EVM message frames.
      */
-    bytes ethereum_hash = 1;
+    repeated ContractAction contract_actions = 1;
 
     /**
-    * A result for an Ethereum Transaction execution, either a create or call.
-    * <p>
-    * This field SHALL contain all of the data produced by the contract
-    * create transaction as well as basic accounting results.
-    */
-    proto.ContractFunctionResult contract_function_result = 2;
-
+     * An ethereum transaction hash value.
+     * <p>
+     * This SHALL be a keccak256 hash of the ethereumData that represents the RLP encoded transaction.
+     */
+    bytes ethereum_hash = 2;    
```

Comment:
Note, this in under consideration for removal. Just need to confirm there's no edge cases of recalculation resulting in different values